### DISCUSSION TITLE
Pouvoir changer le nom de guilde

### DISCUSSION THREAD

**Discussion by Stitch-seagull** [Crownicles/Crownicles#869] [Suggestions fermées]
En étant chef pouvoir change le nom d'une guilde ( avec un cooldown pour éviter les abus )

**Comment by DoctorT1807** [comment]
Sa ressemble à la #580

**Comment by BastLast** [comment]
Je crois que c'est la 3 ème fois que quelqu'un poste cette idée alors qu'elle a déjà été envoyée, faite attention à ne pas faire de duplicata !!!
